src/search/search-bar.tsx

```diff
@@ -111,7 +113,11 @@ export const SearchBar: FC<SearchBarProps> = ({
 			if (inputTyped.length > 0) {
 				const newInputState = [
 					...inputState,
-					...map(inputTyped?.split(' '), (item) => ({ label: item, hasAvatar: false }))
+					...map(inputTyped?.split(' '), (item: string, id: number) => ({
```

Comment:
it can also be define as
`
...map(inputTyped?.split(' '), (label: string, id: number) => ({
						id,
						label,
						hasAvatar: false
					}))
`

Reply:
Almost: I used the shorthand for the _label_ property, but the _id_ still need to be converted to string since the new object has to be a _QueryChip_